pkg/deployments/tasks/20221115-aave-rebalanced-linear-pool/test/test.fork.ts

```diff
@@ -38,7 +38,7 @@ describeForkTest('AaveLinearPoolFactory', 'mainnet', 15225000, function () {
   let poolId: string;
 
   before('run task', async () => {
-    task = new Task('20220817-aave-rebalanced-linear-pool', TaskMode.TEST, getForkedNetwork(hre));
+    task = new Task('20221115-aave-rebalanced-linear-pool', TaskMode.TEST, getForkedNetwork(hre));
```

Comment:
Let's add a test where the rebalancer uses USDT. It'd also be great if we could have a simple test for a reverting rate, but I'm not sure we can easily craft that.